Makefile

```diff
@@ -20,6 +20,7 @@ ALL_SRC := $(shell find . -name "*.go" | grep -v -e vendor \
 
 .PHONY: test
 test: $(COV_REPORT)
+	$(ECHO_V)go test -race -timeout=1s $(PKGS)
```

Comment:
add $(MAKE) $(COV_REPORT) on the next line, it will produce much better output in travis (won't run coveralls when the tests fail_